### ISSUE TITLE
Vessel vanished when going on EVA without obvious reason

### ISSUE THREAD

**Issue by ShyRed** [LunaMultiplayer/LunaMultiplayer#171]
Playing together as 2 players, sometimes a vessel disappears when a player is on EVA.

Previously this was reproduceable when 2nd player started a new vessel. This does not happen anymore. (thank you very much for the fix! Your good work is really appreciated! :smiley: :+1: ).  Instead the vessel now disappears for no obvious reason.

I was about to land on the mun and decided to do a quick EVA for that sweet research points. When going EVA with Bill, suddenly the vessel together with Jebediah vanished.

According to server log it seems that both Avoider and me had the lock for Bill and Jeb and that some backup was in progress. Please notice that Avoider had no manned flight at that point and was working on a drone vessel without crew in VAB.

My KSP logfile only showed a single log entry from LMP:

[WRN 20:30:24.905] [LMP] Adjusted time from: 31073820.4217391 to: 31073851.924843 due to error: -31503.103852272

------------------------------------------------------------------------------

OS: 
Damaskar: Windows 7, 64 Bit
Avoider: Windows 7, 64 Bit

------------------------------------------------------------------------------

KSPVersion: 
1.4.5  (without the history DLC)

------------------------------------------------------------------------------

LMP version (include the build number if you are using a nighly build): 
0.12.48 Build 0.12.48.656 ( https://ci.appveyor.com/project/gavazquez/lunamultiplayer/branch/master/job/lrgjg7nriotyleqd )

------------------------------------------------------------------------------

MODS (add link to download them): 

- Scatterer 0.0331b ( https://github.com/LGhassen/Scatterer/releases/tag/0.0331b )
- Kerbal Engineer Redux v1.1.5.3 ( https://github.com/jrbudda/KerbalEngineer/releases/tag/1.1.5.3 )

------------------------------------------------------------------------------

Expected behaviour: 

Vessels stay with the player, if the player goes on EVA.

------------------------------------------------------------------------------

Current behaviour: 

Vessels sometimes despawn / disappear when the player goes on EVA.

------------------------------------------------------------------------------

Steps to reproduce: 

No idea unfortunately. Here is what we did when it happend:

1. I was about to land on mun with a vessel with Bill and Jeb
2. I went on EVA with Bill.
3. Avoider was in VAB building a drone vessel (no crew).
4. My vessel vanished...

[lmpserver_2018-08-16_18-27-18.log](https://github.com/LunaMultiplayer/LunaMultiplayer/files/2294844/lmpserver_2018-08-16_18-27-18.log)

**Comment by ShyRed** [comment]
Just did a bit more testing. I'm pretty confident that the time sync caused the problem.

I tried the following:
1. Create pod with 2 crew members.
2. Start flight and cheat into orbit around mun
3. Go on EVA
4. Force time sync via LMP

The result is that the kerbal that is holding onto the vessel stops holding onto it after the time sync.

I can imagine that the time sync caused a big enough shift that the vessel ended up smashing into the mun maybe? If that is really the case (I did not get any crash info, Jeb is just "missing in action" for me), could there be a way to disable time syncs while a kerbal is on EVA?
